src/Unleash/Communication/IUnleashApiClient.cs

```diff
@@ -6,7 +6,7 @@ namespace Unleash.Communication
 {
     internal interface IUnleashApiClient
     {
-        Task<FetchTogglesResult> FetchToggles(string etag, CancellationToken cancellationToken);
+        Task<FetchTogglesResult> FetchToggles(string etag, CancellationToken cancellationToken, bool throwOnFail = false);
```

Comment:
public method but on an internal interface. this should be ok?

Reply:
Yeah I think so